sql/core/benchmarks/IntervalBenchmark-results.txt

```diff
@@ -1,29 +1,29 @@
+prepare string w/ interval                          648            721          94          1.5         648.3       1.0X
+prepare string w/o interval                         562            596          49          1.8         562.3       1.2X
+1 units w/ interval                                 568            590          21          1.8         568.5       1.1X
+1 units w/o interval                                522            538          20          1.9         521.7       1.2X
+2 units w/ interval                                 751            754           3          1.3         751.5       0.9X
+2 units w/o interval                                716            723           6          1.4         716.1       0.9X
+3 units w/ interval                                1402           1411          11          0.7        1401.6       0.5X
+3 units w/o interval                               1381           1387           5          0.7        1381.2       0.5X
+4 units w/ interval                                1591           1595           6          0.6        1591.2       0.4X
+4 units w/o interval                               1582           1585           3          0.6        1582.3       0.4X
+5 units w/ interval                                1747           1749           2          0.6        1747.3       0.4X
+5 units w/o interval                               1738           1746          10          0.6        1737.7       0.4X
+6 units w/ interval                                1929           1931           3          0.5        1929.1       0.3X
+6 units w/o interval                               1919           1922           2          0.5        1919.0       0.3X
+7 units w/ interval                                2345           2354           8          0.4        2345.0       0.3X
+7 units w/o interval                               2334           2336           2          0.4        2334.1       0.3X
+8 units w/ interval                                2533           2546          16          0.4        2533.0       0.3X
+8 units w/o interval                               2519           2521           1          0.4        2519.4       0.3X
+9 units w/ interval                                2885           2889           5          0.3        2884.5       0.2X
+9 units w/o interval                               2804           2813          12          0.4        2803.9       0.2X
+10 units w/ interval                               3041           3060          16          0.3        3041.3       0.2X
+10 units w/o interval                              3031           3043          15          0.3        3031.2       0.2X
+11 units w/ interval                               3270           3280           9          0.3        3269.9       0.2X
+11 units w/o interval                              3273           3280           7          0.3        3272.6       0.2X
```

Comment:
Regression by 66%. Here is the JIRA for investigation: https://issues.apache.org/jira/browse/SPARK-30562

Reply:
Hm, regression vs what here... the change should have made the benchmarks faster, but they were also re-run. Is it the new instance type?

Reply:
There are a few commits in interval implementation between 2 re-runs. The regression can be related to those commits but to my changes.  

> Is it the new instance type?

It is the same.